addons/account/static/tests/tours/account_dashboard_setup_bar_tests.js

```diff
@@ -0,0 +1,49 @@
+odoo.define('account.dashboard.setup.tour', function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var tour = require('web_tour.tour');
+
+    var _t = core._t;
+
+    tour.register('account_render_report', {
+        test: true,
+        url: '/web',
+    }, [tour.STEPS.SHOW_APPS_MENU_ITEM,
+    {
+        trigger: '.o_app[data-menu-xmlid="account.menu_finance"]',
+        position: 'bottom',
+        edition: 'community'
+    }, {
+        trigger: '.o_app[data-menu-xmlid="account_accountant.menu_accounting"]',
```

Comment:
Hello,

Minor issue about this task, it introduces the test `account_render_report` in `account` but it actually depends on enterprise `account_accountant`.

So when running the test with enterprise but without `account_accountant` (since it is not a dependency) it fails.

I believe an override should be made directly in `account_accountant` to include the additional steps.

Reply:
@Arcasias Did you fix it ?

Reply:
@seb-odoo thanks for reporting the issue ! The fix has just been merged in 13.0 with 
 the following PRs: https://github.com/odoo/odoo/pull/38945 (com) and https://github.com/odoo/enterprise/pull/6208 (ent)

Reply:
@FlorianGilbert Linked to my request, could interest you.